Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -31,14 +31,24 @@ public struct CustomerCenterConfigData {
 
     public struct Localization {
 
-        let locale: String
-        let localizedStrings: [String: String]
+        public let locale: String
+        public let localizedStrings: [String: String]
```

Comment:
I'm wondering, could we make these private and only provide localizations through the `commonLocalizedString` method?

Reply:
let's do that and expose them if needed

Reply:
keeping them internal for tests though?

Reply:
Sure that's fine!